scripts/upload_artifact_s3.py

```diff
@@ -0,0 +1,38 @@
+import os
+import boto3
+import datetime
+from botocore.exceptions import ClientError
+
```

Comment:
just please specify which environment variables should be set

like in https://github.com/openshift/lightspeed-service/blob/main/scripts/data_collection/download_ols_archive.py#L3

Reply:
100%, thank you!